js/main.js

```diff
@@ -98,7 +139,7 @@ function changeViewNav(event) {
   if (event.target.matches('.nav-link')) {
     var dataViewValue = event.target.getAttribute('data-view');
     data.view = dataViewValue;
-    for (var i = 0; i < $viewList.length; i++) {
+    for (i = 0; i < $viewList.length; i++) {
```

Comment:
you should definitely declare your variable here instead of using a global.

Reply:
Done.